.github/PULL_REQUEST_TEMPLATE/hacktoberfest.md

```diff
@@ -0,0 +1,16 @@
+#### Description of Change
```

Comment:
I don't think this change should be in this PR.

Reply:
Oops, that definitely was a mistake.